src/components/include/include.test.ts

```diff
@@ -22,6 +20,14 @@ const stubbedFetchResponse: Response = {
   clone: sinon.fake()
 };
 
+function delayResolve(resolveValue: string) {
```

Comment:
This was a last minute bug fix. For some reason after the refactors, the event was getting triggered before the listener was attached. It was basically acting synchronous due to the stub for some reason. I have no clue why it only affected this one file. But this was an easy fix to add another "tick" of delay to ensure the listeners are attached before the promise resolves. Would love another set of eyes to see where I may have changed the behavior.

Reply:
In that case, I think this might help simplify things. We can look into _why_ it's needed later.

```js
import { aTimeout } from '@open-wc/testing';

// ...

await aTimeout();
```